src/components/Toasts/Toasts.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react'
+
+import { Props } from './Toasts.types'
+import './Toasts.css'
+
+export default class Toasts extends React.PureComponent<Props> {
```

Comment:
@cazala Should we export ToastPosition from here?, to be able to import it from the root package (`decentraland-ui`)